docs/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Next
 
+# 5.23.3 (2019-04-10)
+
+* Progress page changes to utilize more screen real-estate.
```

Comment:
Throw this under “Next” and itll be auto included in the next release version.

Reply:
oh fancy, thanks.